torchmetrics/functional/text/bert.py

```diff
@@ -465,7 +467,7 @@ def _rescale_metrics_with_baseline(
 def bert_score(
     preds: Union[List[str], Dict[str, Tensor]],
     target: Union[List[str], Dict[str, Tensor]],
-    model_name_or_path: Optional[str] = None,
+    model_name_or_path: str,
```

Comment:
^^ ops, same here :]
```suggestion
    model_name_or_path: Optional[str] = None,
```